packages/trader/src/Assets/Trading/Categories/contract-type-description-video.tsx

```diff
@@ -34,6 +31,7 @@ const ContractTypeDescriptionVideo = ({ selected_contract_type }: { selected_con
             controls
             width={isMobile() ? 328 : 480}
             height={isMobile() ? 184.5 : 270}
+            data-testid='description_video'
```

Comment:
```suggestion
            data-testid={data_testid}
```
type of data_testid:
`data_testid?: string;`
@kate-deriv let's set `data_testid` via props in the test itself. I remember there were discussions in favor of this approach sometime ago :)

Reply:
Sure, thank for noticing